include/mscclpp/npkit/npkit_event.hpp

```diff
@@ -15,4 +15,7 @@
 #define NPKIT_EVENT_EXECUTOR_OP_BASE_ENTRY 0x5
 #define NPKIT_EVENT_EXECUTOR_OP_BASE_EXIT 0x17
 
+#define NPKIT_EVENT_KERNEL_ALLREDUCE_ENTRY 0x6
+#define NPKIT_EVENT_KERNEL_ALLREDUCE_EXIT 0x7
+
```

Comment:
Please put this before `NPKIT_EVENT_EXECUTOR_OP_BASE_ENTRY` and increase `NPKIT_EVENT_EXECUTOR_OP_BASE_ENTRY` and `NPKIT_EVENT_EXECUTOR_OP_BASE_EXIT` by 2 respectively